Bottom: 7d644ac3ce2e522f6f871f6c01119137239cf594
Top:    cd37e742d89248b53973fa9fdf594fe8d07ab8b5
Author: scott snyder <sss@karma>
Date:   2020-02-01 14:34:40 -0500

Refresh of tbb

---

diff --git a/DDDigi/src/DigiKernel.cpp b/DDDigi/src/DigiKernel.cpp
index d62c6694..13445c17 100644
--- a/DDDigi/src/DigiKernel.cpp
+++ b/DDDigi/src/DigiKernel.cpp
@@ -91,7 +91,7 @@ public:
   DigiEventAction*  action = 0;
   Wrapper(DigiContext& c, DigiEventAction* a)
     : context(c), action(a) {}
-  Wrapper(Wrapper&& copy) = delete;
+  Wrapper(Wrapper&& copy) = default;
   Wrapper(const Wrapper& copy) = default;
   Wrapper& operator=(Wrapper&& copy) = delete;
   Wrapper& operator=(const Wrapper& copy) = delete;
@@ -111,7 +111,7 @@ class DigiKernel::Processor {
   DigiKernel& kernel;
 public: 
   Processor(DigiKernel& k) : kernel(k) {}
-  Processor(Processor&& l) = delete;
+  Processor(Processor&& l) = default;
   Processor(const Processor& l) = default;
   void operator()()  const {
     int todo = 1;
@@ -273,7 +273,7 @@ void DigiKernel::submit(const DigiAction::Actors<DigiEventAction>& actions, Digi
   if ( parallel )   {
     tbb::task_group que;
     for ( auto* i : actions )
-      que.run(Wrapper(context, *i));
+      que.run(Wrapper(context, &*i));
     que.wait();
     goto print_stamp;
   }
